contrib/platform/test/com/sun/jna/platform/win32/PsapiUtilTest.java

```diff
@@ -0,0 +1,48 @@
+/* Copyright (c) 2020 Torbjörn Svensson, All Rights Reserved
+ *
+ * The contents of this file is dual-licensed under 2
+ * alternative Open Source/Free licenses: LGPL 2.1 or later and
+ * Apache License 2.0. (starting with JNA version 4.0.0).
+ *
+ * You can freely decide which license you want to apply to
+ * the project.
+ *
+ * You may obtain a copy of the LGPL License at:
+ *
+ * http://www.gnu.org/licenses/licenses.html
+ *
+ * A copy is also included in the downloadable source code package
+ * containing JNA, in file "LGPL2.1".
+ *
+ * You may obtain a copy of the Apache License at:
+ *
+ * http://www.apache.org/licenses/
+ *
+ * A copy is also included in the downloadable source code package
+ * containing JNA, in file "AL2.0".
+ */
+package com.sun.jna.platform.win32;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import java.util.Arrays;
+
+import org.junit.Test;
+
+/**
+ * Applies API tests on {@link PsapiUtil}.
+ *
+ * @author Torbj&ouml;rn Svensson, azoff[at]svenskalinuxforeninen.se
+ */
+@SuppressWarnings("nls")
+public class PsapiUtilTest {
+    @Test
+    public void enumProcesses() {
+        int myPid = Kernel32.INSTANCE.GetCurrentProcessId();
+        int[] pids = PsapiUtil.enumProcesses();
+
+        assertNotNull("List should not be null", pids);
+        assertTrue("List should contain my pid", Arrays.stream(pids).anyMatch(pid -> pid == myPid));
```

Comment:
We retain compatibility with Java 1.6, so as neat as streams and lambdas are, you have to do it the old fashioned way.

Reply:
If you aim to also have the test classes in Java6, I suggest that the Eclipse project is set to have Java6 as the target and the build.xml should also check ./test-classes, not just ./classes.
Anyway, I'll push a fix for this.

Reply:
There are one or two exceptions in the test classes that require Java 8, which is why the project includes a higher compatibility level in the config files, but we otherwise maintain the standard.

Reply:
(I know this because I did the same thing not too long ago!)